src/css/custom.css

```
@@ -153,5 +181,40 @@ main{
   max-width: 900px;
 }
 .footer--dark {
-  background-color: var(--ifm-footer-background-color) !important;
+  background: black !important;
+  padding-top:2rem;
+
+}
+
+.footer__title{
+  font-weight: 600;
+  font-size: 0.9rem;
+  color: var(--text-500)
+
+}
+
+@media (max-width: 996px){
+  .footer {
+    --ifm-footer-padding-horizontal: 1.5rem;
+}
+}
+
+
+/* snippet */
+
+.codeBlockLines_node_modules-\@docusaurus-theme-classic-lib-next-theme-CodeBlock-styles-module{
```

```suggestion
code[class^="codeBlockLines"] {
```
we can simplify in this way